cmd/horcrux/cmd/config.go

```diff
@@ -24,19 +24,17 @@ func configCmd() *cobra.Command {
 
 func initCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "init [chain-id] [chain-nodes]",
+		Use:     "init [chain-nodes]",
 		Aliases: []string{"i"},
```

Comment:
Technically, this should be a v3.0 because we are breaking the external API (which is a cli interface in this case). I recommend going to v3 because it signals to users to be careful of breaking changes.

Reply:
I'm on the fence, but I suppose that's reason enough to go to v3.

Justifications for v2.2.0: existing horcrux users can do an update from v2.1.0 or v2.1.1 with the same config file. Only new users, or users who use config init as part of their deployment startup routine to generate the config file every startup, need to be aware of this change.

Justifications for v3.0.0: CLI API breaking change. Existing horcrux users need to do a non-rolling update (shut down all nodes and start all back up on new version) due to ChainID parameter added to GRPC cosigner API requests. We have done non-rolling updates as minors in the past though.